score/tile/generate.py

```diff
@@ -0,0 +1,91 @@
+import os
+from pathlib import Path
+import shutil
+
+from etl.sources.census.etl_utils import get_state_fips_codes
+
+
+def generate_tiles(data_path: Path):
+
+    # remove existing mbtiles file
+    mb_tiles_path = data_path / "tiles" / "block2010.mbtiles"
+    if os.path.exists(mb_tiles_path):
+        os.remove(mb_tiles_path)
+
+    # remove existing mvt directory
+    mvt_tiles_path = data_path / "tiles" / "mvt"
+    if os.path.exists(mvt_tiles_path):
+        shutil.rmtree(mvt_tiles_path)
+
+    # Merge scores into json
+    # TODO: for this first pass, just merging ACS EJScren indicators
+    #       Per https://github.com/usds/justice40-tool/issues/102
+
+    if os.name == "nt":
+        pwd = "%cd%"
+    else:
+        pwd = "${PWD}"
+
+    # remove existing score json files
+    score_geojson_dir = data_path / "score" / "geojson"
+    files_in_directory = os.listdir(score_geojson_dir)
+    filtered_files = [file for file in files_in_directory if file.endswith(".json")]
+    for file in filtered_files:
+        path_to_file = os.path.join(score_geojson_dir, file)
+        os.remove(path_to_file)
+
+    # join the state shape sqllite with the score csv
+    state_fips_codes = get_state_fips_codes()
```

Comment:
Nit: should we add more logging statements so it's easy to tell where we are in all these various stages?

Reply:
There's never a case where fewer logging statements make sense. 😉